transcom/mymove
# INT-B-18638

## [Agility ticket](tbd)

## Summary

Updated to match 18637

### How to test

1. Access the customer app
2. Login as a new user
3. Create a move with the Wounded Warrior order type
4. Test Wounded Warrior label on customer home page header and move details header
5. Login as a TOO/TIO/SC/QAE/Prime and search for new move
6. Make sure Warrior label is visible within the move queue and on the Move Details Header
